modules/javafx.base/src/main/java/javafx/beans/value/ObservableValue.java

```diff
@@ -272,8 +272,8 @@ default <U> ObservableValue<U> flatMap(Function<? super T, ? extends ObservableV
      * <p>
      * For example:
      * <pre>{@code
-     * ObservableValue<Boolean> condition = new SimpleBooleanProperty(true);
-     * ObservableValue<String> longLivedProperty = new SimpleStringProperty("A");
+     * Property<Boolean> condition = new SimpleBooleanProperty(true);
```

Comment:
A better alternative might be to use a `var` like at line 160 and 191.

Reply:
I think it is fine as is, some extra type information, especially in examples, never hurts.